platform/security/sts/security-sts-ldapclaimshandler/src/main/java/ddf/security/sts/claimsHandler/ClaimsHandlerManager.java

```diff
@@ -230,6 +232,19 @@ protected LDAPConnectionFactory createLdapConnectionFactory(String url, Boolean
         } catch (NumberFormatException ignore) {
         }
 
+        try {
```

Comment:
Also, yes, I realize it is a dupe. However, I don't see it as something that is reusable outside of just these two classes, nor was it enough code to warrant me connecting some sort of dependency to hold just this chunk of code (they share no other common code or dependencies).